compiler/lib/unit_info.mli

```diff
@@ -41,3 +41,7 @@ val prefix : string
 val to_string : t -> string
 
 val parse : t -> string -> t option
+
+val to_sexp : t -> Sexp.t list
```

Comment:
Does this really need to live inside jsoo ? It seems to me that you have everything you need to implement it in a separate module.